torch/csrc/api/src/nn/modules/transformer.cpp

```diff
@@ -13,8 +18,8 @@ TransformerEncoderLayerImpl::TransformerEncoderLayerImpl(
 }
 
 void TransformerEncoderLayerImpl::reset() {
-  // NOTE: reset() is for initializing the model only, call reset() after the model is created
-  // will cause throwing exceptions. Call reset_parameter() if the created model need a reset
+  // NOTE: reset() is for initializing the model only, calling reset() after the model is created
+  // will cause throwing exceptions. Call reset_parameter() if the created model needs a reset
```

Comment:
nit: "will cause throwing exceptions" -> "will cause exceptions to be thrown", or "will throw exceptions"